doc/pyinstaller.1

```diff
@@ -43,7 +43,7 @@ PyInstaller is a program that freezes (packages) Python programs into
 stand\-alone executables, under Windows, GNU/Linux, Mac OS X,
 FreeBSD, OpenBSD, Solaris and AIX.
 Its main advantages over similar tools are that PyInstaller works with
-Python 2.7 and 3.5—3.7, it builds smaller executables thanks to transparent
+Python 2.7 and 3.5—3.8, it builds smaller executables thanks to transparent
```

Comment:
Please remove this change for now - the man pages need syncing anyway, and we can add this in when that get's done.